### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by blakelr** [quantixed/HirstGenerator#1]
Would it be possible to replicate this but with a given image instead of generating dots? Let's say I had a logo that was a triangle type shape, create this automatically with that logo?

**Comment by quantixed** [comment]
Yes, I think it is possible to draw any shape in place of the circles in Igor (the program I used here). It might be easier to do it on another platform.
